python/tests/unit/test_bootstrap.py

```diff
@@ -23,7 +21,7 @@ def _test_bootstrap(yaml, v2, expectations={}):
         if expected is None:
             assert key not in bootstrap
         else:
-            import json
+            pass
```

Comment:
```python
pass

assert key in bootstrap
assert bootstrap[key] == expected
```

may not be what's intended here

Reply:
yup...good catch!

Reply:
Fixed, interestingly. I don't think anything in that file is called at all but I will leave looking into cleaning it up as a separate activity.